src/content/en/updates/_shared/no-new-deprecations.html

```diff
@@ -0,0 +1 @@
+No features have been deprecated in this version of Chrome. Chrome Platform Status provides a [list of deprecated features](https://www.chromestatus.com/features#browsers.chrome.status:%22Deprecated%22) from previous versions of Chrome.
```

Comment:
You can't use markdown in HTML files, please change the markdown link to an `<a href...` link.